rmgpy/data/thermo.py

```diff
@@ -1327,6 +1335,23 @@ def estimateRadicalThermoViaHBI(self, molecule, stableThermoEstimator ):
                 logging.error(molecule)
                 logging.error(molecule.toAdjacencyList())
                 raise
+            
+            # Add the radical interactions which are ignored in the thermo calculation of molecule.
+            if saturatedStruct.isAromatic() and (not saturatedStruct.isAtomInCycle(atom)):
```

Comment:
Thanks for the clarification that parenthesis is unnecessary.  
The 'atom' here is the atom that has the radical site, not the H to be added. So it could be in a cycle.